misc/docker-ci/check.mk

```diff
@@ -93,7 +93,7 @@ _do_check:
 	cmake $(CMAKE_ARGS) -H$(SRC_DIR) -B.
 	time komake $(BUILD_ARGS) all checkdepends
 	if [ -e h2o-fuzzer-http1 ] ; then export $(FUZZ_ASAN); fi; \
-		ulimit -n 1024; \
+		ulimit -n 10000; \
```

Comment:
This was done to allow `t/40redis-session-resumption.t` test to pass, given fixes here: https://github.com/h2o/h2o/commit/4fd069b723e31710948c43ee8bfe68373275695a I wonder if this could be reverted

Reply:
Nice catch! Pushed ecd7f6e that reverts this, let's see how CI goes.

Reply:
CI succeeded, let's stay at 1024.